.travis.yml

```diff
@@ -47,11 +47,13 @@ dist: trusty
 sudo: required
 addons:
    postgresql: "9.4"
-   apt:
-    packages:
-    - mysql-server-5.6
-    - mysql-client-core-5.6
-    - mysql-client-5.6
+   mysql: "5.7"
```

Comment:
@slint Yes we should say what versions Invenio has been tested with. It is however hard to say it now as Invenio is still unstable and I am not sure that all services, i.e. CDS labs, Inspire, Zenodo & B2SHARE, use the same versions.